cmd/livepeer_cli/wizard.go

```diff
@@ -41,6 +42,23 @@ func (w *wizard) readString() string {
 	}
 }
 
+func (w *wizard) readMultilineString() string {
+	ss := make([]string, 1)
+	fmt.Printf("(press enter, then ctrl+d when done) > ")
```

Comment:
We probably shouldn't tell the user to press enter and then ctrl+d if this would add an additional newline to the message that was not originally there.